testframework/src/main/java/net/neoforged/testframework/impl/EventListenerGroupImpl.java

```diff
@@ -6,44 +6,49 @@
 package net.neoforged.testframework.impl;
 
 import java.util.ArrayList;
-import java.util.HashMap;
 import java.util.List;
-import java.util.Map;
 import java.util.function.Consumer;
 import javax.annotation.ParametersAreNonnullByDefault;
 import net.minecraft.MethodsReturnNonnullByDefault;
 import net.neoforged.bus.api.Event;
 import net.neoforged.bus.api.EventPriority;
 import net.neoforged.bus.api.IEventBus;
-import net.neoforged.fml.common.EventBusSubscriber;
 import net.neoforged.testframework.Test;
 import org.jetbrains.annotations.ApiStatus;
 
 @ApiStatus.Internal
 @ParametersAreNonnullByDefault
 @MethodsReturnNonnullByDefault
 public class EventListenerGroupImpl implements Test.EventListenerGroup {
-    private final Map<EventBusSubscriber.Bus, EventListenerCollectorImpl> collectors = new HashMap<>();
+    private final EventListenerCollectorImpl mod = new EventListenerCollectorImpl(),
+            game = new EventListenerCollectorImpl();
 
     @Override
-    public EventListenerCollectorImpl getFor(EventBusSubscriber.Bus bus) {
-        return collectors.computeIfAbsent(bus, it -> new EventListenerCollectorImpl());
+    public EventListenerCollector mod() {
+        return mod;
     }
 
-    public void unregister(Map<EventBusSubscriber.Bus, IEventBus> buses) {
-        collectors.forEach((bus, col) -> col.unregisterAll(buses.get(bus)));
+    @Override
+    public EventListenerCollector forge() {
```

Comment:
Should this be neoforge instead of forge?

Reply:
That would be an unnecessary cosmetic breaking change which I did not want to make.

Reply:
I’m on my phone so I am probably missing something but I don’t see `forge` method in this class’s removal line. This looks like a brand new addition

Reply:
Oh wait it’s an override. I see